Traceback (most recent call last):
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 64, in wait_till_response_arrives
    message = q.get(timeout=timeout)
  File "D:\Lib\queue.py", line 212, in get
    raise Empty
_queue.Empty

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus\browser_decorator.py", line 201, in run_task
    result = func(driver, data)
  File "c:\Users\Admin\Downloads\Code\OOPFinalProject\Project\MyTest\try_surfing.py", line 32, in download_html_pages
    driver.google_get(link, bypass_cloudflare=True)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 2230, in google_get
    self.get_via(
    ~~~~~~~~~~~~^
        link,
        ^^^^^
    ...<4 lines>...
        timeout=timeout,
        ^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 2204, in get_via
    self.detect_and_bypass_cloudflare()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 1397, in detect_and_bypass_cloudflare
    bypass_if_detected(self)
    ~~~~~~~~~~~~~~~~~~^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\solve_cloudflare_captcha.py", line 273, in bypass_if_detected
    solve_full_cf(driver)
    ~~~~~~~~~~~~~^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\solve_cloudflare_captcha.py", line 158, in solve_full_cf
    checkbox = get_checkbox(iframe)
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\solve_cloudflare_captcha.py", line 171, in get_checkbox
    return iframe.select("input", wait=None)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\driver.py", line 257, in select
    elem = self._elem.query_selector(selector, wait)
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\element.py", line 877, in query_selector
    self.update()
    ~~~~~~~~~~~^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\element.py", line 311, in update
    doc =  self._tab.send(cdp.dom.get_document(-1, True))
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 290, in send
    return self.perform_send(cdp_obj, _is_update, wait_for_response)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 310, in perform_send
    result = wait_till_response_arrives(self.queue, tx_id , MAX_WAIT)
  File "C:\Users\Admin\Downloads\Code\OOPFinalProject\Project\.venv\Lib\site-packages\botasaurus_driver\core\connection.py", line 68, in wait_till_response_arrives
    raise TimeoutError("Response not received")
TimeoutError: Response not received
